concepts/controls.mdx

```diff
@@ -116,3 +116,18 @@ For the full list of parameters, check out the [full SDK reference](/sdks/framew
 - [Zod Schema](https://zod.dev/) - A TypeScript-first schema declaration and validation library.
 - [JSON Schema](/recipes/json-schema) - The most popular schema language for defining JSON data structures.
 
+## Using Variables
+
+To facilitate the use of variables in the control schema, enclose the variable name in double curly braces using the `{{variableName}}` syntax. For example, `{{subscriber.firstName | capitalize}}` will be dynamically replaced with the actual value of the subscriber's first name at runtime.
+You can use variables in any step control value, whether set by the developer or within the Novu Dashboard UI. To facilitate this, the Novu Dashboard UI offers auto-completion for variables. Simply start typing `{{` to view a list of all available variables.
+
+--TODO: insert gif of autocomplete in dashboard ---
```

Comment:
Thinking maybe its not needed, not sure if I should add one or not 🤔

Reply:
@ainouzgali Is variables supported only in step controls?